README.md

```diff
@@ -86,6 +86,11 @@ You would need set some npm configuration, this is optional.
 $ npm set registry http://localhost:4873/
 ```
 
+For one-off commands or to avoid setting the registry globally:
+```bash
+NPM_CONFIG_REGISTRY=http://localhost:4873 npm i
```

Comment:
Hi @alex-dixon, interesting en var. Could you share from where this is coming from ?

Reply:
https://docs.npmjs.com/misc/config#environment-variables

Reply:
Thanks !!